client-java/controller-api/src/main/java/org/evomaster/client/java/controller/api/dto/SutInfoDto.java

```diff
@@ -49,7 +49,15 @@ public enum OutputFormat {
         KOTLIN_JUNIT_4,
         KOTLIN_JUNIT_5,
         JS_JEST,
-        CSHARP_XUNIT
+        CSHARP_XUNIT;
+
+        public boolean isJava(){
+            return this.name().startsWith("JAVA");
+        }
+
+        public boolean isJavaOrKotlin(){
+            return this.name().startsWith("JAVA") || this.name().startsWith("KOTLIN");
+        }
     }
```

Comment:
wasn't this code done in another branch? am i looking at a branch that pull in from another branch that has not been merged yet? in that case, i should wait until the other branch is merged before looking into this one